src/osx/textentry_osx.cpp

```diff
@@ -175,7 +175,7 @@ void wxTextEntry::SetInsertionPointEnd()
 
 long wxTextEntry::GetInsertionPoint() const
 {
-    long begin, end ;
+    long begin = 0, end = 0;
```

Comment:
@Dan-Korn Do you remember what problem did this fix? It looks like `GetSelection()` always fills in the provided parameters and there are still a few places where it's called with them being uninitialized. IMO we should either initialize them everywhere, if really necessary, or drop this commit if it isn't.